src/FITSIO.jl

```diff
@@ -147,6 +147,23 @@ include("header.jl")  # FITSHeader methods
 include("image.jl")  # ImageHDU methods
 include("table.jl")  # TableHDU & ASCIITableHDU methods
 
+"""
+    libcfitsio_version(version=fits_get_version()) -> VersionNumber
```

Comment:
For this particular function, the optional argument may not be documented, it's just a trick to make it possible testing the parsing of version number, I don't expect anybody will use it.